jfrog-applications/SUMMARY.md

```diff
@@ -5,8 +5,10 @@
 ## JFrog Applications
 
 * [JFrog CLI](jfrog-cli/README.md)
-  * [Download and Insall](jfrog-cli/get-started/download-and-insall/install.md)
-  * [Usage](jfrog-cli/get-started/download-and-insall/usage.md)
+  * [Download and Install](jfrog-cli/get-started/download-and-install/README.md)
+    * [Download and Install](jfrog-cli/get-started/download-and-install/install.md)
```

Comment:
I'm noy sure that two `Download and Install` titles one after the other is the right thing to do.